Formula/matplotlib.rb

```diff
@@ -1,101 +1,60 @@
-class NoExternalPyCXXPackage < Requirement
-  fatal false
-
-  satisfy do
-    !quiet_system "python3", "-c", "import CXX"
-  end
-
-  def message
-    <<~EOS
-      *** Warning, PyCXX detected! ***
-      On your system, there is already a PyCXX version installed, that will
-      probably make the build of Matplotlib fail. In python you can test if that
-      package is available with `import CXX`. To get a hint where that package
-      is installed, you can:
-          python3 -c "import os; import CXX; print(os.path.dirname(CXX.__file__))"
-      See also: https://github.com/Homebrew/homebrew-python/issues/56
-    EOS
-  end
-end
-
 class Matplotlib < Formula
+  include Language::Python::Virtualenv
+
   desc "Python 2D plotting library"
   homepage "https://matplotlib.org"
-  url "https://files.pythonhosted.org/packages/10/5f/10c310c943f29e67976dcc26dccf9305a5a9bc7483e631ee74a0f95aa5b2/matplotlib-2.2.5.tar.gz"
-  sha256 "a3037a840cd9dfdc2df9fee8af8f76ca82bfab173c0f9468193ca7a89a2b60ea"
-  head "https://github.com/matplotlib/matplotlib.git"
-
-  bottle do
-    root_url "https://linuxbrew.bintray.com/bottles-bio"
-    sha256 "609aaf08da967a0a7729bfb9502f29f9303617154612195f0b2100c4dd2797a7" => :catalina
-    sha256 "d118af398ad5341b4eb16f3e44195ceae12295311a819b6e0c6b14e8558c2c52" => :x86_64_linux
-  end
```

Comment:
No need to delete the `bottle do` block. It gets updated automatically by CI. PR looks good otherwise. Thanks!